compilation error on FreeBSD

any thoughts?

```
=== ERROR while installing sexplib.110.01.00 =====
opam-version         1.1.0
os                   freebsd
command              gmake
path                 /usr/home/hannes/.opam/4.01.0/build/sexplib.110.01.00
compiler             4.01.0
exit-code            2
env-file             /usr/home/hannes/.opam/4.01.0/build/sexplib.110.01.00/sexplib-72969-e7966f.env
stdout-file          /usr/home/hannes/.opam/4.01.0/build/sexplib.110.01.00/sexplib-72969-e7966f.out
stderr-file          /usr/home/hannes/.opam/4.01.0/build/sexplib.110.01.00/sexplib-72969-e7966f.err
# stdout ###
...[truncated]

./setup.exe -build 
ocamlfind ocamlopt unix.cmxa -I /home/hannes/.opam/4.01.0/lib/ocaml/ocamlbuild /home/hannes/.opam/4.01.0/lib/ocaml/ocamlbuild/ocamlbuildlib.cmxa -linkpkg myocamlbuild.ml /home/hannes/.opam/4.01.0/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
ocamlfind ocamldep -package bigarray -modules lib/conv.mli > lib/conv.mli.depends
ocamlfind ocamldep -package bigarray -modules lib/sexp.mli > lib/sexp.mli.depends
ocamlfind ocamldep -package bigarray -modules lib/sexp_intf.ml > lib/sexp_intf.ml.depends
ocamlfind ocamldep -pp 'cpp -undef -traditional -Isyntax' -package bigarray -modules lib/pre_sexp.ml > lib/pre_sexp.ml.depends
+ ocamlfind ocamldep -pp 'cpp -undef -traditional -Isyntax' -package bigarray -modules lib/pre_sexp.ml > lib/pre_sexp.ml.depends
Command exited with code 2.
gmake[1]: Leaving directory `/usr/home/hannes/.opam/4.01.0/build/sexplib.110.01.00'
# stderr ###
W: Cannot find source file matching module 'sexplib_num' in library sexplib_num
W: Cannot find source file matching module 'sexplib' in library sexplib
E: Failure("Command ''/home/hannes/.opam/4.01.0/bin/ocamlbuild' lib/sexplib.cma lib/sexplib.cmxa lib/sexplib.a lib/sexplib.cmxs num/lib/sexplib_num.cma num/lib/sexplib_num.cmxa num/lib/sexplib_num.a num/lib/sexplib_num.cmxs unix/lib/sexplib_unix.cmx unix/lib/sexplib_unix.cmo syntax/pa_sexp_conv.cma syntax/pa_sexp_conv.cmxa syntax/pa_sexp_conv.a syntax/pa_sexp_conv.cmxs top/sexplib_top.cma top/sexp...[truncated]
gmake[1]: *** [build] Error 1
```

--- a/_tags
+++ b/_tags
@@ -1,6 +1,6 @@
 # OASIS_START
 # OASIS_STOP
-<lib/pre_sexp.ml>: pp(cpp -undef -traditional -Isyntax)
+<lib/pre_sexp.ml>: pp(cpp -undef -traditional -Isyntax -Wno-invalid-pp-token)
 <lib_test/*.ml{,i}>: syntax_camlp4o
 <syntax/pa_sexp_conv.ml>: syntax_camlp4o
 "top/sexplib_install_printers.ml": I(+compiler-libs)

